graph/schema.graphqls

```diff
@@ -159,12 +167,7 @@ type Vehicle implements Node {
   """
   A Relay-style connection listing any active privilege grants on this vehicle.
   """
-  privileges(
-    first: Int
-    after: String
-    last: Int
-    before: String
```

Comment:
We have to preserve these in order to follow Relay. The filter should be in addition to these.